app/__init__.py

```diff
@@ -16,13 +17,19 @@
 
 
 def null_to_dash(value):
+    """Filter that converts string values that are "null" or Nones to dash"""
     if value == "null":
         return "-"
     if value is None:
         return "-"
     return value
 
 
+def clean_tags(text):
+    """Filter that removes ALL HTML tags that are not <mark>"""
```

Comment:
Annoying that opensearch doesnt sanitize results like elastcsearch does: https://www.elastic.co/guide/en/app-search/current/sanitization-guide.html


[Maybe?/might not be worth] It could be worth, making "mark" a variable so we can pass the sam variable to this function and to the function that builds up the highlighting?

Reply:
also, wha happens if our docs contain mark tags?

Reply:
I think we'd need to use the variable approach, specify the highlight marker to be a uuid then

`.replace(f{"<{highlight_marker_uuid}>", "<mark>").replace(f"</{highlight-unique-12345}>", "</mark>")`
when using n the template.